src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -607,6 +608,103 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def createProject(projectName: RawlsBillingProjectName, billingAccount: RawlsBillingAccountName, projectTemplate: ProjectTemplate): Future[Unit] = {
+    val credential = getBillingServiceAccountCredential
+
+    val cloudResManager = getCloudResourceManager(credential)
+    val billingManager = getCloudBillingManager(credential)
+    val computeManager = getComputeManager(credential)
+
+    val projectResourceName = s"projects/${projectName.value}"
+    for {
+      // create the project
+      project <- retryWhen500orGoogleError(() => {
```

Comment:
retryWithRecoverWhen500orGoogleError executes the recover before the retry whereas this approach does not but either way works here because conflict status codes are not retried